index.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Shanul Huda Ansari</title>
+  <title>Shanul Huda & Sultan Ansari</title>
```

Comment:
please add your first name only .
```suggestion
  <title>Shanul Huda & Sultan Ansari</title>
```
```suggestion
  <title>Shanul Huda & Sultan Ansari</title>
```

Reply:
okay.